src/controllers/blocks/BlocksExtrinsicsController.ts

```diff
@@ -0,0 +1,66 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+
+import { BlocksService } from '../../services';
+import { INumberParam } from '../../types/requests';
+import AbstractController from '../AbstractController';
+
+export default class BlocksExtrinsicsController extends AbstractController<BlocksService> {
+	constructor(api: ApiPromise) {
+		super(api, '/blocks/:blockId/extrinsics', new BlocksService(api));
+		this.initRoutes();
+	}
+
+	protected initRoutes(): void {
+		this.safeMountAsyncGetHandlers([
+			['/:extrinsicIndex', this.getExtrinsicByExtrsinsicIndex],
+		]);
+	}
+
+	/**
+	 *
+	 * @param _req Express Request
+	 * @param res Express Response
+	 */
+	private getExtrinsicByExtrsinsicIndex: RequestHandler<INumberParam> = async (
```

Comment:
```suggestion
	private getExtrinsicByIndex: RequestHandler<INumberParam> = async (
```
…seems less of a tongue twister?